console/Command.php

```diff
@@ -109,17 +116,23 @@ protected function _init() {
 		if (!is_object($this->request) || !$this->request->params) {
 			return;
 		}
-		$this->response = $this->_config['response'];
 
-		if (!is_object($this->response)) {
-			$this->response = $this->_instance('response', $this->response);
-		}
 		$default = array('command' => null, 'action' => null, 'args' => null);
 		$params = array_diff_key((array) $this->request->params, $default);
 
 		foreach ($params as $key => $param) {
+			if($key == 'no-color') {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/14819903/11846519/e1a6e97e-a410-11e5-8280-c40fb1c73f49.png)

![image](https://cloud.githubusercontent.com/assets/14819903/11846531/f0216c5e-a410-11e5-802d-144b27fa165d.png)


Multi words are not automatically mapped to noColor at the moment no.